backend/Makefile

```diff
@@ -17,9 +17,8 @@ $(BINARIES): $(BUILD_DIR)/hashi-ui-%: $(BUILD_DIR) bindata_assetfs.go
 .PHONY: install
 install:
 	@go mod download
-	@go get github.com/go-bindata/go-bindata/...
```

Comment:
If I remember correctly, I couldn't get this to work other than whats in the deleted section now

Reply:
That's weird. It worked for me on both my `$GOPATH` and also on an empty one:

```
% go version
go version go1.13.3 darwin/amd64
% make --version
GNU Make 3.81
Copyright (C) 2006  Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.

This program built for i386-apple-darwin11.3.0
% pwd
/Volumes/Development/hashi-ui
% unset GOPATH
% export GOPATH=$(pwd)/gopath
% export PATH=$PATH:$(pwd)/gopath/bin
% mkdir $GOPATH
% ls -l $GOPATH
% make clean
=> cleaning ...
/Library/Developer/CommandLineTools/usr/bin/make -j -C backend clean
=> cleaning backend ...
rm -rf /Volumes/Development/hashi-ui/backend/build
rm -f bindata_assetfs.go
/Library/Developer/CommandLineTools/usr/bin/make -j -C frontend clean
=> cleaning ...
rm -rf build
% make build
/Library/Developer/CommandLineTools/usr/bin/make -j -C backend build
go: finding github.com/DataDog/datadog-go v2.2.0+incompatible
[...]
=> packaging assets ...
go-bindata-assetfs -prefix ../frontend ../frontend/build/...
=> building /Volumes/Development/hashi-ui/backend/build/hashi-ui-darwin-amd64 ...
GOOS=darwin GOARCH=amd64 CGO_ENABLED=0 go build -o /Volumes/Development/hashi-ui/backend/build/hashi-ui-darwin-amd64 -ldflags "-X main.GitCommit=v1.1.2-8-ge7edfb2+CHANGES"
/Library/Developer/CommandLineTools/usr/bin/make -j -C backend test
==> Running test...
=== RUN   TestConfig_correctDefaulValues
--- PASS: TestConfig_correctDefaulValues (0.00s)
PASS
ok  	github.com/jippi/hashi-ui/backend/config	(cached)
[...]
% ls backend/build
hashi-ui-darwin-amd64
% ls $GOPATH/bin
go-bindata		go-bindata-assetfs
```